src/sections/selectNode.tsx

```diff
@@ -43,14 +43,18 @@ function Section1() {
     dispatch(authActions.useNodeData({ ip, apiKey }));
     dispatch(authActions.addNodeData({ ip, apiKey: saveApiKey ? apiKey : '' }));
     dispatch(authActionsAsync.login({ ip, apiKey }));
-    dispatch(nodeActionsAsync.getInfo());
+    dispatch(
+      nodeActionsAsync.getInfoThunk({ apiToken: apiKey, apiEndpoint: ip })
```

Comment:
should rename local state here so it does not handle the variables with name: `apiKey` and `ip`

Reply:
Good point, changed